src/CloudIoTCoreMQTTClient.h

```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ * Copyright 2018 Google
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#ifndef CloudIoTCoreMQTTClient_h
+#define CloudIoTCoreMQTTClient_h
+
+#include <Arduino.h>
+#include <WiFiClientSecure.h>
```

Comment:
This will not build on MKR1000 because of this include.

Reply:
A reminder that I really need to get CI in place so you can see these issues upon creating a CL.

Reply:
Yeah, definitely we need to setup a CI build for this project. After this PR I'll study more about the `PlatformIO` integration and apply here, I use it for local development, should be difficult to use it in a CI environment.